### DISCUSSION TITLE
What is Renovate?

### DISCUSSION THREAD

**Discussion by OnkelTem** [renovatebot/renovate#27314] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

In all my Gitlab js repositories I've got MRs that add `renovate.json`, seems like they were created automatically.
For some times I was postponing a moment to learn about that thing and today I've finally opened your GitHub repository. 
I read the first page and still don't have clear understanding of what this is all about.

Could you please explain it in simple words?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
"Get automated Pull Requests to update your dependencies"

**Reply by OnkelTem** [reply]
Thanks for the response, Rhys.

Could you please elaborate a little more?

I believe under dependencies you mean package.json dependencies, isn't? Does this software update them automatically? Isn't it dangerous? Even package-lock.json is there for preserving specific deps to guarantee reproducibility. And Renovate is going to replace package-lock.json and go further and fetch new version without asking user?

What precise problem does this software address?

**Reply by rarkins** [reply]
Sorry, I don't have time

**Reply by OnkelTem** [reply]
Ok, then I'll wait for someone who knows what this software is doing and why.
